css/style.css

```diff
@@ -98,3 +98,308 @@
   margin-left: 24px;
   color: #505f79;
 }
+
```

Comment:
- [ ] **Optional:** I recommend you increase the `min-width` of your `.main-content` class between 425 and 480 to cover all sizes of mobile devices. At the moment, when mobile devices that exceed 375 are used to view your website, there is a whitish edge.
![image](https://user-images.githubusercontent.com/27709832/149299378-5daf35bb-7402-4db2-b4df-35ca1d102983.png)

Reply:
Increasing min-width of .main-content in media? Like this?
 @media (max-width:425) and (min-width:480px) {
    .main-content {
      // rest of code
    }
}
I did that but things messed up but however this is just optional on Figma width is 375px so I really don't know this one. So if you can just comment back on this one and approve it since it's optional but I want the answer.

Reply:
I mean like this I exchanged vaues but still not working.. So can you please let me know did You mean to do it like this

@media (min-width:425) and (max-width:480px) {
.main-content {
// rest of code
}
}